client/components/LeftDrawer/index.jsx

```diff
@@ -0,0 +1,266 @@
+import List from '@material-ui/core/List';
+import Divider from '@material-ui/core/Divider';
+import IconButton from '@material-ui/core/IconButton';
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import LinkIcon from '@material-ui/icons/Link';
+import { connect } from 'react-redux';
+import { toggleMenu as reduxToggleMenu } from '@reducers/ui';
+import Radio from '@material-ui/core/Radio';
+
+const drawerWidth = 275;
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    display: 'flex',
+  },
+  appBar: {
+    transition: theme.transitions.create(['margin', 'width'], {
+      easing: theme.transitions.easing.sharp,
+      duration: theme.transitions.duration.leavingScreen,
+    }),
+  },
+  appBarShift: {
+    width: `calc(100% - ${drawerWidth}px)`,
+    marginLeft: drawerWidth,
+    transition: theme.transitions.create(['margin', 'width'], {
+      easing: theme.transitions.easing.easeOut,
+      duration: theme.transitions.duration.enteringScreen,
+    }),
+  },
+  menuButton: {
+    marginRight: theme.spacing(2),
+  },
+  hide: {
+    display: 'none',
+  },
+  drawer: {
+    width: drawerWidth,
+    flexShrink: 0,
+  },
+  drawerPaper: {
+    width: drawerWidth,
+    backgroundColor: '#2A404E',
+  },
+  content: {
+    flexGrow: 1,
+    padding: theme.spacing(3),
+    transition: theme.transitions.create('margin', {
+      easing: theme.transitions.easing.sharp,
+      duration: theme.transitions.duration.leavingScreen,
+    }),
+    marginLeft: -drawerWidth,
+  },
+  share: {
+    marginBottom: '100px',
+    paddingLeft: '25px',
+  },
+  listItem: {
+    paddingTop: '15px',
+    paddingBottom: '15px',
+    height: '24px',
+  },
+  listItemTitle: {
+    paddingLeft: '28px',
+  },
+}));
+
+const PersistentDrawerLeft = ({ menuIsOpen, toggleMenu }) => {
+  // TODO ADD FUNCTIONALITY
+  const [selectedMapStyleValue, setMapStyleValue] = React.useState('Point Map');
+  const [selectedMapModeValue, setMapModeValue] = React.useState('Dark');
+  const [selectedDataColorScheme, setDataColorScheme] = React.useState('Original');
+  const [selectedBoundariesValue, setBoundariesValue] = React.useState('None');
+
+  const handleChangeMapStyle = event => {
+    setMapStyleValue(event.target.value);
+  };
+
+  const handleChangeMapMode = event => {
+    setMapModeValue(event.target.value);
+  };
+
+  const handleChangeDataColorScheme = event => {
+    setDataColorScheme(event.target.value);
+  };
+
+  const handleChangeBoundaries = event => {
+    setBoundariesValue(event.target.value);
+  };
+
+  const classes = useStyles();
+  const theme = useTheme();
+
+  const onClickShare = () => {
+    // TODO ADD FUNCTIONALITY
+  };
+
+  const escFunction = e => {
+    e.preventDefault();
```

Comment:
it looks like this is preventing the chrome devtools keyboard shortcut from working. a couple options that fixed it, hopefully without causing any side effects:

1. move e.preventDefault inside the `if` block
2. change the eventListener to 'keypress' instead of 'keydown'

keypress fires after keydown, so maybe that's somehow messing up the shortcut?

Reply:
Will look into it